### PR TITLE
Add SSSE3 sad asm

### PR DESCRIPTION
**Pull request by KyleSiefring**
Rust is beating me on larger blocks, though.

### CODE REVIEW

File: src/me.rs [hunk 1]
```diff
@@ -14,12 +14,77 @@ use partition::*;
...
+    dst_stride: libc::ptrdiff_t
+  ) -> u32;
+
+  fn rav1e_sad_32x32_ssse3(
+    src: *const u16, src_stride: libc::ptrdiff_t, dst: *const u16,
+    dst_stride: libc::ptrdiff_t
+  ) -> u32;
+}
+
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```
Comment (lu-zero) [inline src/me.rs]:
```suggestion
#[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), not(windows)))]
```

You forgot about windows :)

File: src/me.rs [hunk 2]
```diff
@@ -14,12 +14,77 @@ use partition::*;
...
+  }
+  return sum;
+}
 
 #[inline(always)]
 pub fn get_sad(
   plane_org: &PlaneSlice, plane_ref: &PlaneSlice, blk_h: usize,
   blk_w: usize
 ) -> u32 {
+  #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```
Comment (lu-zero) [inline src/me.rs]:
```suggestion
#[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), not(windows)))]
```
Same here

**Comment by lu-zero** [comment]
Looks fine, do you have some benchmark numbers?

**Comment by tdaede** [comment]
Ping, can you add the needed comments?

**Pull request merged**